TikiClient/Classes/TikiClient.swift

```diff
@@ -31,7 +33,8 @@ public class TikiClient {
     ///   - company: The legal information of the company.
     public static func initialize(userId: String, completion: @escaping (String?) -> Void) {
         if(config == nil){
-            fatalError("Config is nil")
+            completion("Config is nil")
```

Comment:
It shoudl throw an error or call a onError callback

Reply:
done